ir/shared/src/main/scala/org/scalajs/ir/Trees.scala

```diff
@@ -1110,6 +1122,19 @@ object Trees {
     val tpe = AnyType
   }
 
+  sealed case class LinkTimeProperty(name: String)(val tpe: Type)(
```

Comment:
The `Tag`s must be strictly additive, and therefore added at the end. However, for everything else, we try to preserve some semblance of logical order. Move this up after all the `Literal`s, perhaps? And likewise move all the `case LinkTimeProperty`s to be in the same "position"?